python/sglang/srt/layers/quantization/awq.py

```diff
@@ -191,6 +190,7 @@ def apply(
         pack_factor = self.quant_config.pack_factor
         out_shape = x.shape[:-1] + (qweight.shape[-1] * pack_factor,)
         reshaped_x = x.reshape(-1, x.shape[-1])
+        from sgl_kernel import awq_dequantize
```

Comment:
MUST NOT import in forward phase. And we shoud move it to `__init__`

Reply:
The reason I move the import here is that `skl_kernel` is not available to other devices and importing it directly will cause error even if awq is not used. 

Anyway, I added an `is_cuda` guard at the top, do you think it is ok?